source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
```

Comment:
I just don't think we use the `override` keyword for `abstract` functions anywhere else. Better have @davidhesselbom say something about this.